mkt/extensions/tests/test_views.py

```diff
@@ -148,6 +149,11 @@ def setUp(self):
         self.list_url = reverse('api-v2:extension-list')
         self.user = UserProfile.objects.get(pk=2519)
 
+    def tearDown(self):
+        super(TestExtensionViewSetPost, self).tearDown()
+        # Explicitely delete the Extensions to clean up leftover files.
```

Comment:
Yeah, but for the upload tests it's kinda painful to fully mock the filesystem so I didn't do it. I did it for the rest of the tests though, like reject/publish.